scrape_dbPedia.js

```diff
@@ -0,0 +1,98 @@
+const http = require('http');
+var express = require('express');
```

Comment:
`var` vs `const` vs `let`. Are they variables, constants or scope-specific? Be consistent with the use of these!